utils.h

```diff
@@ -0,0 +1,9 @@
+#ifndef UTILS_H
+#define UTILS_H
+
+#include "mkmimo.h"
```

Comment:
Ahh I just tried this, and unfortunately the Inputs and Outputs type declarations in mkmimo.h depend on the Buffer type declaration. It looks like we have two options: we can include mkmimo.h in utils/buffer.h or vise versa, or we can just combine the files entirely.